frontend/src/layout/panel-layout/ProjectTree/ProjectTree.tsx

```diff
@@ -0,0 +1,592 @@
+                                            >
+                                                New Folder
+                                            </LemonButton>
+                                        ) : null}
+                                        {item.record?.path ? (
+                                            <LemonButton
+                                                onClick={() => item.record?.path && rename(item.record.path)}
+                                                fullWidth
+                                                size="small"
+                                            >
+                                                Rename
+                                            </LemonButton>
+                                        ) : null}
+                                        {item.record?.path ? (
+                                            <LemonButton
+                                                onClick={(e) => {
+                                                    e.stopPropagation()
+                                                    handleCopyPath(item.record?.path)
+                                                }}
+                                                fullWidth
+                                                size="small"
+                                            >
+                                                Copy Path
+                                            </LemonButton>
+                                        ) : null}
+                                        {item.record?.created_at ? (
+                                            <LemonButton
+                                                onClick={(e) => {
+                                                    e.stopPropagation()
+                                                    deleteItem(item.record as unknown as FileSystemEntry)
+                                                }}
+                                                fullWidth
+                                                size="small"
+                                            >
+                                                Delete
+                                            </LemonButton>
+                                        ) : null}
+                                    </>
+                                }
+                            />
+                        ),
+                        identifier: item.record?.path || 'more',
+                    }
+                }}
+            />
+        </PanelLayoutPanel>
+    )
+
+    // return (
+    //     <>
+    //         <nav
+    //             className={clsx('flex flex-col max-h-screen min-h-screen relative w-[320px] border-r border-primary')}
+    //             ref={containerRef}
+    //         >
+    //             <div className="flex justify-between p-1 bg-surface-tertiary">
+    //                 <ProjectDropdownMenu />
```

Comment:
I'm not the biggest fan of committing in a bunch of uncommented code. Why not just remove it?